### PR TITLE
Prevent retries when there is no backup

### PR DESCRIPTION
**Pull request by ahmetmircik**
closes https://github.com/hazelcast/hazelcast/issues/11308

### CODE REVIEW

**Comment by Donnerbart** [comment]
Can we have the test from the linked comment in the issue as part of the PR?

File: hazelcast/src/main/java/com/hazelcast/map/impl/operation/EvictBatchBackupOperation.java [hunk 1]
```diff
@@ -68,6 +70,17 @@ public void run() {
         }
     }
 
+    @Override
+    public ExceptionAction onInvocationException(Throwable throwable) {
+        if (throwable instanceof WrongTargetException) {
```
Comment (tombujok) [inline hazelcast/src/main/java/com/hazelcast/map/impl/operation/EvictBatchBackupOperation.java]: Looks good. Could we just have a short comment here?

Comment (ahmetmircik) [reply hazelcast/src/main/java/com/hazelcast/map/impl/operation/EvictBatchBackupOperation.java]: done

**Comment by jerrinot** [comment]
run-lab-run

**Comment by Donnerbart** [comment]
It looks like the commit message is unrelated to the PR:
> Converted cacheName --> cacheId and userGivenCacheName --> cacheName

This should be fixed for a useful commit history.

**Comment by ahmetmircik** [comment]
@Donnerbart fixed

**Pull request merged**